src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -12,7 +12,9 @@ public class Phone {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+
+    // allow either a blank string, whitespaces (will be trimmed) or require all numbers and at least 3 digits long
```

Comment:
I know the requirement for the phone numbers in AB3 was to be at least 3 digits long but I'm wondering if we should enforce it to be exactly 8 digits  (and have first digit 8/9)? Assuming the wedding planners are based in Singapore... just not sure if others testing the application in the future will pick this up as a bug